### PR TITLE
#499 Show Last Item Review Name and Date 

### PR DESCRIPTION
**Pull request by thestrugglingblack**
# Ticket Number
#266 

# Description
The purpose of this PR is to show to the user the name and date of the last individual who made a *status* change to the item in the current queue.

# Areas of Change
- `database/item.js` : update schema to support `lastModifiedBy` and `lastModifiedDate`
- `lib/helper/item-route-validators.js` : Added `lastModifiedBy` and `lastModifiedDate` to be accepted attributes
- `migrations/20180309161931-add_last_modified_by_last_modified_date_to_item.js` : update the DB for those two new columns in the `Item` table in Postgres
- ` routes/item.js` : added `updateItem` method to handle `lastModifiedBy` and `lastModifiedDate`

# Test
* Pull branch `feature/tf-499_item-last-modified`
* Startup a local instance of Postgres
* Run `npm start`
* Startup a local instance of to-fix internal and pull in branch `feature/tf-499_item-last-modified`
* Go to an item
* Change the status 
* Review that item detail and see if there was a change in the date and name. 
* Repeat again if needed.

_If the above items do not work as stated please contact me and leave a comment on what happened_

### CODE REVIEW

**Comment by kepta** [comment]
@ZuriHunter can you use the project's prettier configuration.

**Comment by thestrugglingblack** [comment]
@kepta you mean that configs from .eslintrc file?

**Comment by kepta** [comment]
Prettier https://github.com/osmlab/to-fix-backend/blob/master/package.json#L11

**Comment by thestrugglingblack** [comment]
@kepta My settings in atom was not picking up prettier correctly.

**Review by kepta — APPROVED** [review]
LGTM!

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
+    "prestart": "npm run db:migrate",
```
Comment (danmactough) [inline package.json:8]: 😎

**Review by danmactough — APPROVED** [review]
looks good!

**Pull request merged**